src/teleop/gui/src/components/MastGimbalControls.vue

```diff
@@ -10,12 +10,16 @@
 <script>
 import ROSLIB from "roslib";
 
-//keycodes
+// Keycodes
 const W = 87;
 const A = 65;
 const S = 83;
 const D = 68;
 
+// Motor power levels
+const ROTATION_PWR = 1.0;
```

Comment:
can we make this configurable in the teleop.yaml?

Reply:
addressed